### DISCUSSION TITLE
CR2 RAW files left behind

### DISCUSSION THREAD

**Discussion by ababcock** [TheLastGimbus/GooglePhotosTakeoutHelper#270] [I NEED HELP]
I am impressed with the work this tool did to stamp and organize my JPEGs, but I have about 50GB of CR2 Canon RAW files left behind in my original Photos takeout folder.  Many of the RAW files are the only copy, meaning I didn't set my Canon camera to shoot JPEG+RAW.
<img width="730" alt="Screenshot 2023-12-28 at 8 38 17 AM" src="https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/4739241/80f0b040-dee0-429f-88a2-d204b3fd69b9">

**Comment by TheLastGimbus** [comment]
Transferred to issue :+1:

**Comment by XBadPlayerX** [comment]
Thanks for the work you have done on this!

I hope you can add this file type and maybe others like .png. Maybe that works already? Probably does, you are smart.

**Comment by Hengy2016** [comment]
Hi, 
also to add my thanks for a great tool, 
and add a request for Sony ARW file support similar to the CR2 request
